ngx-cookie-service dependency broken (regression)

##### **Overview of the issue**
https://github.com/jhipster/generator-jhipster/pull/16906 broke release 7.4.0 due to unsatisfied `@angular/common@13.0.0` dependency.

##### **Motivation for or Use Case**
Should work. Deploy tests to detect this?

##### **Reproduce the error**
To reproduce just create a new empty angular project with `session` auth and `websocket` and it explodes with this:

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: app-1@0.0.1-SNAPSHOT
npm ERR! Found: @angular/common@12.2.12
npm ERR! node_modules/@angular/common
npm ERR!   @angular/common@"12.2.12" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @angular/common@"^13.0.0" from ngx-cookie-service@13.0.0
npm ERR! node_modules/ngx-cookie-service
npm ERR!   ngx-cookie-service@"13.0.0" from the root project
```
##### **Related issues**

<!-- Has a similar issue been reported before? Please search both closed & open issues -->

##### **Suggest a Fix**
Either fast track the Angular 13 upgrade or revert `ngx-cookie-service` back to version 12.

##### **JHipster Version(s)**
7.4.0
regression

##### **JHipster configuration**

INFO! Using JHipster version installed globally
Welcome to the JHipster Information Sub-Generator

##### **JHipster Version(s)**

```
app-1@0.0.1-SNAPSHOT O:\GitHub\test
`-- (empty)

```


##### **JHipster configuration, a `.yo-rc.json` file generated in the root folder**


<details>
<summary>.yo-rc.json file</summary>
<pre>
{
  "generator-jhipster": {
    "authenticationType": "session",
    "cacheProvider": "ehcache",
    "clientFramework": "angularX",
    "serverPort": "8080",
    "serviceDiscoveryType": false,
    "skipUserManagement": false,
    "withAdminUi": true,
    "baseName": "app1",
    "buildTool": "maven",
    "databaseType": "sql",
    "devDatabaseType": "h2Disk",
    "enableHibernateCache": true,
    "enableSwaggerCodegen": false,
    "enableTranslation": true,
    "jhiPrefix": "jhi",
    "languages": ["en"],
    "messageBroker": false,
    "prodDatabaseType": "postgresql",
    "searchEngine": false,
    "testFrameworks": [],
    "websocket": "spring-websocket",
    "enableGradleEnterprise": false,
    "gradleEnterpriseHost": "",
    "jhipsterVersion": "7.4.0",
    "packageFolder": "com/mycompany/myapp",
    "packageName": "com.mycompany.myapp",
    "clientPackageManager": "npm",
    "dtoSuffix": "DTO",
    "entitySuffix": "",
    "reactive": false,
    "clientTheme": "none",
    "clientThemeVariant": "",
    "applicationType": "monolith",
    "applicationIndex": 0,
    "entities": ["A", "B", "C"],
    "skipCheckLengthOfIdentifier": false,
    "skipFakeData": false,
    "blueprints": [],
    "otherModules": [],
    "pages": [],
    "nativeLanguage": "en",
    "creationTimestamp": 1637451574642,
    "rememberMeKey": "YourJWTSecretKeyWasReplacedByThisMeaninglessTextByTheJHipsterInfoCommandForObviousSecurityReasons",
    "devServerPort": 4200,
    "lastLiquibaseTimestamp": 1637451754000
  }
}

</pre>
</details>


##### **JDL for the Entity configuration(s) `entityName.json` files generated in the `.jhipster` directory**

<details>
<summary>JDL entity definitions</summary>

<pre>
entity A
entity B
entity C
dto A, B, C with mapstruct
paginate A, B, C with infinite-scroll
service A, B, C with serviceClass

</pre>
</details>


##### **Environment and Tools**

openjdk version "11.0.12" 2021-07-20
OpenJDK Runtime Environment Temurin-11.0.12+7 (build 11.0.12+7)
OpenJDK 64-Bit Server VM Temurin-11.0.12+7 (build 11.0.12+7, mixed mode)

git version 2.29.2.windows.1

node: v16.13.0

npm: 8.1.4


No change to package.json was detected. No package manager install will be executed.

<details>
<summary>JDL definitions</summary>
<pre>
application {
  config {
    baseName app1
    authenticationType session
    websocket spring-websocket
  }
  entities A, B, C
  dto * with mapstruct
}

entity A
entity B
entity C

paginate * with infinite-scroll
</pre>
</details>

- [x] Checking this box is mandatory (this is just to show you read everything)

<!-- Love JHipster? Please consider supporting our collective:
👉  https://opencollective.com/generator-jhipster/donate -->

--- a/generators/client/templates/angular/package.json
+++ b/generators/client/templates/angular/package.json
@@ -8,7 +8,7 @@
     "@ng-bootstrap/ng-bootstrap": "10.0.0",
     "@ngx-translate/core": "13.0.0",
     "@ngx-translate/http-loader": "6.0.0",
-    "ngx-cookie-service": "13.0.0",
+    "ngx-cookie-service": "12.0.3",
     "ngx-infinite-scroll": "10.0.1",
     "ngx-webstorage": "8.0.0",
     "rxjs": "6.6.7",

